content/enterprise/_index.md

```diff
@@ -39,17 +39,17 @@ The Enterprise Edition is free for organizations with under $1 million US dollar
 You can download the Enterprise Edition from Dockerhub by following the [installation instructions]({{< relref "server/overview.md" >}}) described in this documentation.
 
 # Where do I download the Open Source Edition?
-There is no official distribution of the Open Source Edition. You can [build](https://github.com/drone/drone/blob/master/BUILDING_OSS) the Open Source Edition from source using the oss build tags. 
+There is no official distribution of the Open Source Edition. You can [build](https://github.com/harness/drone/blob/master/BUILDING_OSS) the Open Source Edition from source using the oss build tags. 
 
 ```
-$ go build -tags "oss nolimit" github.com/drone/drone/cmd/drone-server
+$ go build -tags "oss nolimit" github.com/harness/drone/cmd/drone-server
```

Comment:
this currently won't work:
https://github.com/harness/drone/blob/6b0a6d4f7ea089108b7761d0c21bc622d273c7f8/go.mod#L1

We still need to use `github.com/drone/drone/cmd/drone-server`

Reply:
reverted the change, many thanks !